### ISSUE TITLE
Update Homebrew Failed

### ISSUE THREAD

**Issue by Aryanvkl** [umts/strap#84]
I attempted to setup strap and it failed to update Homebrew with the error Can't Create 'brew update'.

**Comment by werebus** [comment]
Hey @Aryanvkl, can you provide a bit more information? At the very least, the output of your `strap.sh` run - I might also ask for the `--debug` output.

I _suspect_ that the issue here is that you are taking over someone else's laptop that is already _mostly_ set up. But Homebrew operates on the assumption that your laptop only has one admin-ish user and that one user (in this case the old owner of your laptop) is the owner of the Homebrew install.

**Comment by Aryanvkl** [comment]
Of course. 

Updating Homebrew: Error: Can't create `brew update` lock in /usr/local/var/homebrew/locks!
Fix permissions by running:
  sudo chown -R avakil-af /usr/local/var/homebrew
!!! Updating Homebrew: FAILED
!!! Run '/Users/avakil-af/Downloads/strap.sh --debug' for debugging output.
!!! If you're stuck: file an issue with debugging output at:
!!!   https://github.com/umts/strap/issues/new

**Comment by werebus** [comment]
Yeah, I'm feeling better about my hypothesis. Can you tell me what `brew doctor` outputs?

**Comment by Aryanvkl** [comment]
Sure. Its the following:

Error: Can't create `brew vendor-install ruby` lock in /usr/local/var/homebrew/locks!
Fix permissions by running:
  sudo chown -R avakil-af /usr/local/var/homebrew
Error: Failed to upgrade Homebrew Portable Ruby!

**Comment by werebus** [comment]
Alright, I'm going to recommend that you [uninstall Homebrew](https://github.com/Homebrew/install/tree/master?tab=readme-ov-file#uninstall-homebrew) and try again:

```bash
sudo /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/uninstall.sh)"
```

(If it's been a while, you _might_ have to re-run "Make me an Admin" first)

**Comment by Aryanvkl** [comment]
Ok. If I am not mistaken this command is for uninstalling it. So, for reinstall, do I follow the same process I did last time?

**Comment by werebus** [comment]
It seems like you've gotten past your specific issue. But I'm going to leave this open because we should provide some guidance in the script output  to new devs in this situation.
